### PR TITLE
CADC_12085 Updated cleanup.sh and DeleteAction to delete middleware jobs

### PR DESCRIPTION
**Pull request by yeunga**

### CODE REVIEW

File: deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml [hunk 1]
```diff
@@ -40,7 +40,7 @@ spec:
           value: "ivo://cadc.nrc.ca/gms?skaha-users"
         - name: skaha.adminsgroup
           value: "ivo://cadc.nrc.ca/gms?skaha-admins"
-        image: images.canfar.net/skaha-system/skaha:0.9.14
+        image: images.canfar.net/skaha-system/skaha:0.9.16
```
Comment (hjeeves) [inline deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml:43]: Why jump 2 minor version numbers? (14 - 16?)

File: deployment/k8s-config/kustomize/overlays/keel-dev/skaha/skaha-tomcat-deployment.yaml [hunk 2]
```diff
@@ -18,7 +18,7 @@ spec:
           value: "ivo://cadc.nrc.ca/gms?skaha-admins"
         - name: skaha.sessionexpiry
           value: "345600"
-        image: images-rc.canfar.net/skaha-system/skaha:0.9.14
+        image: images-rc.canfar.net/skaha-system/skaha:0.9.16
```
Comment (hjeeves) [inline deployment/k8s-config/kustomize/overlays/keel-dev/skaha/skaha-tomcat-deployment.yaml:21]: as other version change - why the 2-step?

File: skaha/VERSION [hunk 3]
```diff
@@ -1,4 +1,4 @@
 ## deployable containers have a semantic and build tag
 # semantic version tag: major.minor
 # build version tag: timestamp
-TAGS="0.9.14 $(date -u +"%Y%m%dT%H%M%S")"
+TAGS="0.9.16 $(date -u +"%Y%m%dT%H%M%S")"
```
Comment (hjeeves) [inline skaha/VERSION:4]: same question here on version number.

**Review by hjeeves — COMMENTED** [review]
Only question is about the version number jump. Otherwise it looks fine.

File: deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml (hunk 1 above)
Comment (hjeeves) [reply deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml:43]: Understanding is there's a code change in the hopper ahead of this one. :)

**Review by hjeeves — APPROVED** [review]

File: deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml (hunk 1 above)
Comment (yeunga) [reply deployment/k8s-config/kustomize/base/skaha/skaha-tomcat-deployment.yaml:43]: Yes, please refer to CADC-12073 PR.

File: deployment/k8s-config/kustomize/overlays/keel-dev/skaha/skaha-tomcat-deployment.yaml (hunk 2 above)
Comment (yeunga) [reply deployment/k8s-config/kustomize/overlays/keel-dev/skaha/skaha-tomcat-deployment.yaml:21]: Please refer to CADC-12073 PR.

File: skaha/VERSION (hunk 3 above)
Comment (yeunga) [reply skaha/VERSION:4]: Please refer to CADC-12073 PR.

**Review by brianmajor — APPROVED** [review]

**Pull request merged**